lib/node/Creator.js

```diff
@@ -14,22 +14,20 @@ const opFns = {
   '^': 'pow',
 };
 
-const operatorNode = (op, fn, args) => {
+const operatorNode = (op, fn, args, implicit) => {
   return {
     type: 'OperatorNode',
     op: op,
     fn: fn,
     args: args,
+    implicit: implicit,
   };
 };
 
 const NodeCreator = {
   operator (op, args, implicit=false) {
     if (op in opFns) {
-      if (op === '-' && args.length === 1) {
-        debugger;
```

Comment:
I'm pretty sure I put the `debugger` there in a previous diff.  Sorry about the individual diffs being so messy.  I'll probably end up squashing everything in the end.